apis/Google.Devtools.AspNet/Google.Devtools.AspNet.IntegrationTests/CloudErrorReporting/CloudErrorReportingExceptionLoggerIntregrationTest.cs

```diff
@@ -29,48 +28,42 @@ namespace Google.Devtools.AspNet.IntegrationTests
 {
     public class CloudErrorReportingExceptionLoggerIntregrationTest
     {
-        private const string ProjectEnvironmentVariable = "TEST_PROJECT";
         private static string ProjectId;
         private static string TestId;
 
         public CloudErrorReportingExceptionLoggerIntregrationTest()
         {
-            ProjectId = Environment.GetEnvironmentVariable(ProjectEnvironmentVariable);
-            if (string.IsNullOrEmpty(ProjectId))
-            {
-                throw new InvalidOperationException(
-                    $"Please set the {ProjectEnvironmentVariable} environment variable before running tests");
-            }
-            TestId = DateTime.UtcNow.ToString("'test'_yyyyMMddTHHmmssfff", CultureInfo.InvariantCulture);
+            ProjectId = Utils.GetProjectIdFromEnvironment();
+            TestId = Utils.GetTestId();
         }
 
         [Fact]
-        public void ErrorReportingTest()
+        public async void ErrorReportingTest()
         {
-            // Create a test server and make an http request.
-            using (TestServer server = TestServer.Create<ErrorReportingApplication>())
-            {
-                server.HttpClient.GetAsync("").Wait();
+            Task<ErrorStatsServiceClient> clientTask = ErrorStatsServiceClient.CreateAsync();
 
-                // Create a request that will filter on the TestId which is set to the service and version.
-                ListGroupStatsRequest request = new ListGroupStatsRequest
+            // Create a test server and make an http.
+            TestServer server = TestServer.Create<ErrorReportingApplication>();
```

Comment:
So why don't we need to dispose of this any more? (It may be fine - I know nothing about TestServer.)